scripts/fast-reboot

```diff
@@ -188,7 +196,7 @@ sync
 
 # Reboot: explicity call Linux native reboot under sbin
 echo "Rebooting to $NEXT_SONIC_IMAGE..."
-exec /sbin/reboot
+exec /sbin/reboot --no-wtmp --no-wall
```

Comment:
With `--no-wtmp --no-wall`,  what is the reboot time improvement?   I observed reboot command taking around 15seconds to print out fist message, but Pavel mentioned he only saw ~1 second delay.

Reply:
I believe it is a minor minor improvement. No harm.

Reply:
@pavel-shirshov for @jipanyang 's question

Reply:
also we should send wall message, it should not be time consuming at all. Not necessary to strip down this linux feature.